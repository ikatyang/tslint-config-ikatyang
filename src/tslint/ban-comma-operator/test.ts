// good

console.log(1, 2)

// bad

console.log((1, 2));
