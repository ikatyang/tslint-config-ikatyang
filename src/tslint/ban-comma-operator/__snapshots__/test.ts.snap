// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

console.log(1, 2)

// bad

console.log((1, 2));
             ~~~~    [Do not use comma operator here because it can be easily misunderstood or lead to unintended bugs.]

`;
